mne/epochs.py

```diff
@@ -3080,14 +3081,14 @@ def _segment_raw(raw, segment_length=1., verbose=None, **kwargs):
         Length of each segment in seconds. Defaults to 1.
     verbose: bool
         Whether to report what is being done by printing text.
-    **kwargs
-        Any additional keyword arguments are passed to ``Epochs`` constructor.
-
+    preload: bool
```

Comment:
```suggestion
    %(preload)s
    %(verbose)s
```
Can use auto-fillers (`%(something)s`) here and also maintain the same order in this list that there is above

Also please add a section below `Returns` like:
```
    Notes
    -----
    .. versionadded:: 0.20